lotus/utils.py

```diff
@@ -1,6 +1,10 @@
+import base64
+from io import BytesIO
 from typing import Callable
 
 import pandas as pd
+import qwen_vl_utils
```

Comment:
Do we need this as a dependency? I don't think its a well maintained project

Reply:
got rid of it